packages/flutter/lib/src/widgets/form.dart

```diff
@@ -256,6 +257,11 @@ class FormField<T> extends StatefulWidget {
   /// autovalidates, this value will be ignored.
   final bool autovalidate;
 
+  /// If true, indicates that the widget is enabled and if autovalidation is true
```

Comment:
First I would describe what this does, then I would follow up with the details about autovalidation:

```suggestion
  /// Whether the widget/form? is able to receive user input.
  ///
  /// Defaults to true. If [autovalidation] is also true, the field will be validated if enabled
  /// is also true. Likewise, if enabled is false the field will not be validated regardless of
  /// [autovalidate]
  ///
```

Reply:
Much better worded than me! Thanks!